hash.go

```diff
@@ -382,3 +384,67 @@ func (db *RoseDB) HIncrBy(key, field []byte, incr int64) (int64, error) {
 	}
 	return valInt64, nil
 }
+
+// HRandfield return a random field from the hash value stored at key, when called with just
+// the key argument. If the provided count argument is positive, return an array of distinct
+// fields. If called with a negative count, the behavior changes and the command is allowed
+// to return the same field multiple times.
+func (db *RoseDB) HRandfield(key []byte, count interface{}) ([][]byte, error) {
```

Comment:
I see the comamnd HRandField in redis,  it has three params: key, count, and withValues.
so we can define the metod like this: `HRandField(key []byte, count int, withValues bool)`
if count is zero, return directly.

Reply:
I guess that in the current state of rosedb, optional arguments are basically unimplemented.
For example, `WITHSCORES` and `LIMIT` in `ZRange`.
It is possible to implement them at this time, but it may be less confusing to implement them all together at a later time.

Although not related to this PR, there are also other expressions for arguments such as `(1`, `+inf`, etc., and I think that these should be addressed in a later phase of the project.

What do you think?